src/renderer/dx/DxRenderer.hpp

```diff
@@ -209,6 +213,6 @@ namespace Microsoft::Console::Render
 
         [[nodiscard]] D2D1_COLOR_F _ColorFFromColorRef(const COLORREF color) noexcept;
 
-        [[nodiscard]] static DXGI_RGBA s_RgbaFromColorF(const D2D1_COLOR_F color) noexcept;
+        [[nodiscard]] static constexpr DXGI_RGBA s_RgbaFromColorF(const D2D1_COLOR_F color) noexcept;

```

Comment:
constexprs' bodies must live in the header, right?

Reply:
Uh, if shared across a module. If it's in the same module, it's OK. But I'll put it up here as it's better to not end up with a future linking nightmare and reduce the cognitive load.